core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -1,4 +1,36 @@
 <script type="text/ng-template" id="skins/Conversation">
+  <!--
+    Off-screen preview of the next card in order to pre-determine the target
+    height for the card content transition animation.
+  -->
+  <div class="conversation-skin-future-tutor-card" aria-hidden="true">
```

Comment:
Just wondering, is it possible for this to reuse the TutorCard directive somehow? That way, you'd face fewer problems with skew in the future.